src/interfaces/IMetaMorpho.sol

```diff
@@ -31,6 +31,13 @@ interface IOwnable {
 interface IMetaMorphoBase {
     /// @notice The address of the Morpho contract.
     function MORPHO() external view returns (IMorpho);
+
+    /// @notice OpenZeppelin decimals offset used by the ERC4626 implementation.
+    /// @dev Calculated to be max(0, 18 - underlyingDecimals).
+    /// @dev When equal to zero (<=> token decimals >= 18), the protection against the donation front-running attack on
+    /// empty vault is low (see https://docs.openzeppelin.com/contracts/4.x/erc4626#inflation-attack). To protect
```

Comment:
```suggestion
    /// empty vault is low (see https://docs.openzeppelin.com/contracts/5.x/erc4626#inflation-attack). To protect
```
because metamorpho uses the 5.0.0 version of OZ contracts.
I checked and there is no big difference (biggest one is probably the `NOTE` in the code example)